k9mail/src/main/java/com/fsck/k9/ui/messageview/MessageContainerView.java

```diff
@@ -190,8 +191,7 @@ public boolean onMenuItemClick(MenuItem item) {
                                 if (inlineImage) {
                                     attachmentCallback.onSaveAttachment(attachmentViewInfo);
                                 } else {
-                                    //TODO: Use download manager
-                                    new DownloadImageTask(getContext()).execute(uri.toString());
+                                    new DownloadManager.Request(uri);
```

Comment:
Did you test this? I doubt just creating a request will do anything. You'll have to call `DownloadManager.enqueue()`.